### DISCUSSION TITLE
My main account's e-mail address has been deleted

### DISCUSSION THREAD

**Discussion by MiliaKlein** [community/community#124865] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

I created this account using an educational e-mail temporally provided by a university, unbeknownst to the date of its deletion, I didn't start the e-mail migration process and now I am locked out of it, as it keeps sending a verification code to this e-mail - Two-Step verification is not enabled. 

Is retrieving my account still possible? I even have GitHub Desktop linked to it.

**Comment by ebndev** [comment, marked as answer]
Hi @MiliaKlein,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
